main.tf

```diff
@@ -43,13 +43,100 @@ resource "aws_key_pair" "auth" {
 
 resource "aws_s3_bucket" "acme_main" {
   bucket = "test-bucket"
-  acl    = "public-read"
+  acl    = "private"
   versioning {
-    enabled = false
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure VPC flow logging is enabled in all VPCs**
 
**Vulnerability**
Ensure VPC flow logging is enabled in all VPCs
 
**Remediation**
VPC Flow Logs provide visibility into network traffic that traverses the VPC and can be used to detect anomalous traffic or insight during security workflows. Perform the following to determine if VPC Flow logs is enabled: Via the Management Console: 1. Sign into the management console. 2. Select Services then VPC. 3. In the left navigation pane, select Your VPCs. 4. Select a VPC. 5. In the right pane, select the Flow Logs tab. 6. If no Flow Log exists, click Create Flow Log. 7. For Filter, select Reject. 8. Enter in a Role and Destination Log Group. 9. Click Create Log Flow. 10. Click on CloudWatch Logs Group. Additional Reference: CIS Amazon Web Services Foundations Benchmark v1.1.2, https://d0.awsstatic.com/whitepapers/compliance/AWS_CIS_Foundations_Benchmark.pdf

source: accurics-onprem